app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -444,7 +463,7 @@ export const ContentForm: React.FC<IContentFormProps> = ({
                           <FormikText name="page" label="Page" />
                         </Row>
                       </Show>
-                      <Show visible={isSnippetForm(contentType)}>
+                      <Show visible={!isImageForm(contentType) && isSnippetForm(contentType)}>
```

Comment:
This doesn't make sense to me.  `isSnippetForm(...)` should never equal `isImageForm(...)`

Reply:
Updated the legacy logic and so should remove the confusion here, thanks